app/helpers/application_helper.rb

```diff
@@ -9,6 +9,42 @@ def service_name_by_id(id)
     end
   end
 
+  def job_title_name 
```

Comment:
Let's follow the naming convention we've already got in this file for the qualifications, so this and the role helper would be `accepted_job_titles` and `accepted_roles` :)